control/responsible_role.go

```diff
@@ -1,15 +1,16 @@
 package control
 
 import (
-	"github.com/jbowtie/gokogiri/xml"
 	"errors"
+	"fmt"
 	"regexp"
 	"strings"
-	"fmt"
+
+	"github.com/jbowtie/gokogiri/xml"
```

Comment:
FYI that this is [go-plus](https://atom.io/packages/go-plus) making these kinds of changes automatically. I'm not _quite_ that OCD 😉